tests/test_checkers.py

```diff
@@ -7,14 +7,11 @@
 Tests for `inicheck.checkers` module.
 """
 
-import unittest
-
 import inicheck
 from inicheck.checkers import *
 from inicheck.config import MasterConfig, UserConfig
 
-
-class TestCheckers(unittest.TestCase):
+class TestCheckers():
```

Comment:
This should raise a Pep8 warning with two empty lines before a class.
Did you run Pep8 against the test suite?

Reply:
Good catch, I didn't. I will though.